src/qt/utilitydialog.cpp

```diff
@@ -58,7 +58,8 @@ HelpMessageDialog::HelpMessageDialog(QWidget *parent, bool about) :
         ui->helpMessage->setVisible(false);
     } else {
         setWindowTitle(tr("Command-line options"));
-        QString header = "Usage:  bitcoin-qt [command-line options]                     \n";
+        const QString uri_info = wallet_enabled ? QStringLiteral(" [URI]\n\nURI must follow BIP 21") : QString();
+        const QString header(QLatin1String("Usage:  bitcoin-qt [options]") + uri_info + QLatin1Char('\n'));
```

Comment:
nit: well, if care was taken to use lean `QLatin1String`, including `<QStringBuilder>` and using `%` instead of `+` would make it even beter. Using `QStringBuilder` avoids that quadratic re-copying of string buffer with every `+`.

https://wiki.qt.io/Using_QString_Effectively#QStringBuilder_:_Fast_QString_concatenation

https://doc.qt.io/qt-5/qstring.html#more-efficient-string-construction

Reply:
[Updated](https://github.com/bitcoin-core/gui/pull/227#issuecomment-786746698).